config/routes.rb

```diff
@@ -1,6 +1,16 @@
 Rails.application.routes.draw do
 
+  resources :users
```

Comment:
Why are users not in the locale scope? The login, registration, etc. should also be internationalized.